deployment/streaming/scripts/configurations.json

```diff
@@ -0,0 +1,88 @@
+[
```

Comment:
Is this needed?

Reply:
Used [here](https://github.com/azavea/osmesa/blob/baca909e376116350fbb0cf60e32889a9194f0b3/deployment/streaming/scripts/batch-process.sh#L15).  Perhaps it's not needed?  I may have just included this as a test.  I don't entirely recall.

Reply:
When I've run things in the past, I haven't needed this, fwiw. It looks like it may be GT boilerplate for EMR.

Reply:
It's probably not needed.  I was throwing whatever at the wall to get things to run to completion, so I think I can drop it now that things are all sussed out.